docs/es-overview.asciidoc

```diff
@@ -26,13 +26,13 @@ image::images/workflow.png[Elastic Security workflow]
 
 Here's an overview of the flow and its components:
 
-* Data is shipped from your hosts to {es} via beat modules and the Elastic
-Endpoint Security agent integration:
-** <<install-endpoint, {es-sec-endpoint}>> - Elastic agent integration that
+* Data is shipped from your hosts to {es} through the following ways:
+** <<install-endpoint, {es-sec-endpoint}>>: Elastic agent integration that
 protects your hosts <<malware-prevention, against malware>> and ships these data sets:
 ***  *Windows*: Process, network, file, DNS, registry, DLL and driver loads,
 malware security detections
 *** *Linux/macOS*: Process, network, file
+** {integrations-docs}[{integrations}]: Integrations are a streamlined way to connect your data to the {stack}. Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB, as well as many generic input types like log files.
```

Comment:
Small quibble, but the data isn't being connected to the Stack, it's being sent to or ingested by Elastic.
```suggestion
** {integrations-docs}[{integrations}]: Integrations are a streamlined way to send your data to the {stack}. Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB, as well as many generic input types like log files.
```